packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -1511,6 +1512,57 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         );
       });
 
+      withOverriddenEnvsInMochaTest(
+        {
+          GAS_PRICE_TINY_BAR_BUFFER: 2000000000000,
+          MAX_GAS_ALLOWANCE_HBAR: 100,
+        },
+        () => {
+          it('should execute "eth_sendRawTransaction" and pays the total amount of the fees on behalf of the sender', async function () {
+            const balanceBefore = await relay.getBalance(accounts[2].wallet.address, 'latest', requestId);
+
+            const transaction = {
+              type: 1,
+              chainId: Number(CHAIN_ID),
+              nonce: await relay.getAccountNonce(accounts[2].wallet.address, requestId),
+              gasPrice: 0,
+              gasLimit: Constants.MAX_TRANSACTION_FEE_THRESHOLD,
+              data: '0x' + '00'.repeat(Constants.CONTRACT_CODE_SIZE_LIMIT),
+            };
+            const signedTx = await accounts[2].wallet.signTransaction(transaction);
+            const transactionHash = await relay.sendRawTransaction(signedTx, requestId);
+            const info = await mirrorNode.get(`/contracts/results/${transactionHash}`, requestId);
+            expect(info).to.have.property('contract_id');
+            expect(info.contract_id).to.not.be.null;
+            expect(info).to.have.property('created_contract_ids');
+            expect(info.created_contract_ids.length).to.be.equal(1);
+
+            const balanceAfter = await relay.getBalance(accounts[2].wallet.address, 'latest', requestId);
+            expect(balanceAfter).to.be.equal(balanceBefore);
+
+            const transaction2 = {
```

Comment:
isn't this a different test case? If yes, we should split it.

Reply:
The test case seems the same to me, I'm just testing both legacy (type 0) and modern (type 1/2) transactions work correctly with this configuration. Do you prefer I split this in two?

Reply:
Done.